recipe/meta.yaml

```diff
@@ -19,13 +19,15 @@ build:
   number: 0
   # numpy 1.20.0 no longer supports Python 3.6: https://numpy.org/doc/stable/release/1.20.0-notes.html
   # "The Python versions supported for this release are 3.7-3.9, support for Python 3.6 has been dropped"
-  # numpy 1.21.x set Python upper bound <3.11, see https://github.com/numpy/numpy/commit/1e8d6a83985f3191c63963414981743adc4353cf
   skip: True  # [(blas_impl == 'openblas' and win) or py<38]
   force_use_keys:
     - python
 
 requirements:
   build:
+    - python      # added for testing
+    - cython      # added for testing
+    - setuptools  # added for testing
```

Comment:
I don't understand why we need to add these `build` requirements "for testing". Unless you are cross-building, `python` and co. should not needed as build requirements.

Reply:
There were some issues with missing dependencies on the package. The additional dependencies have been removed.